addons/event_sale/models/event_ticket.py

```diff
@@ -91,7 +91,7 @@ def _compute_price_reduce_taxinc(self):
 
     @api.depends('product_id.active')
     def _compute_sale_available(self):
-        inactive_product_tickets = self.filtered(lambda ticket: not ticket.product_id.active)
+        inactive_product_tickets = self.filtered(lambda ticket: not ticket.sudo().product_id.active)
```

Comment:
Why not putting sale_available compute_sudo ? That way we ensure people can read it if they can read the event.